src/index.ts

```diff
@@ -86,8 +84,11 @@ async function main() {
   };
   updateUnitSelection(data[21]);
 
-  const tableFn = (units: Unit[]) =>
+  const tableFn = async (units: Unit[]) => {
```

Comment:
Why make it async?

Reply:
I saw a small difference in speed improvement